splice_spark2/src/main/scala/com/splicemachine/nsds/kafka/KafkaUtils.scala

```diff
@@ -13,14 +28,14 @@ import scala.collection.JavaConverters._
 object KafkaUtils {
   def messageCount(bootstrapServers: String, topicName: String): Long = {
     val props = new Properties
-    val groupId = "spark-consumer-s2s-ku"
+    val groupId = "spark-consumer-nsdsk-ku"
```

Comment:
this should be a const that is used in both messageCount functions. Or have a helper function to remove the copied code from both

Reply:
Addressed in commit [e0977a2](https://github.com/splicemachine/spliceengine/pull/3622/commits/e0977a26b345440115cc0c9d97236d05412ac59b).